docs/csharp/asynchronous-programming/index.md

```diff
@@ -1,73 +1,67 @@
 ---
 title: Asynchronous programming
-description: An overview of the C# language support for asynchronous programming using async, await, Task, and Task<T>
-ms.date: 09/28/2023
+description: Explore an overview of the C# language support for asynchronous programming by using async, await, Task, and Task<T>.
+ms.date: 03/10/2025
 ---
 # Asynchronous programming with async and await
 
-The [Task asynchronous programming model (TAP)](task-asynchronous-programming-model.md) provides an abstraction over asynchronous code. You write code as a sequence of statements, just like always. You can read that code as though each statement completes before the next begins. The compiler performs many transformations because some of those statements may start work and return a <xref:System.Threading.Tasks.Task> that represents the ongoing work.
```

Comment:
I think TAP is a fairly well-known acronym for this type of async programming, so might be good to keep it.

Reply:
The TAP acronym is already applied to the pattern. I thought it would be confusing to reuse it here in a different way.